### DISCUSSION TITLE
Export CSS and JS generated by storybook

### DISCUSSION THREAD

**Discussion by Loic57** [storybookjs/storybook#14022] [Help]
**Is your feature request related to a problem? Please describe**
I'm using Storybook in a HTML/CSS/Vanilla JS mode since a good month and I'm happy with all the things I accomplished. My goal was to create a toolkit (bunch of components as button, headings, badge, card, footer, ...) and It works very well. The problem I met at the end of the project is for the export part. What I did in storybook is just static html code. Now I need to give all my work to a PHP developer who works on Typo3 (a CMS as wordpress/drupal). But I don't know how to give it to him my work.

**Describe the solution you'd like**
What I would have wanted is a command like "npm run extract" which would take all my .scss files and mix them into one file style.css. Same for the JS. Some components I created have a bit of JS like "onclick addclass", "onscroll removeclass", ... and I would need one "scripts.js" file

**Describe alternatives you've considered**
In "storybook-static" folder generated by "npm run build" I can see a lot of js files, maybe there is one file which could help me ? 

**Are you able to assist to bring the feature to reality?**
No, because I'm just a UI designer who have knowledges in HTML/CSS and a bit of JS, all the rest is too complicated for me 😅

**Additional context**
---

Thank you for your assistance

**Comment by shilman** [comment]
The way Storybook typically works is that you create an NPM package of your components and publish it, and then users of that package can simply install it.

As a designer, if you don't want to go through the overhead of figuring out how to do that, or want to maintain such a package, a much lighter-weight handoff would be to simply give him your repository. Chances are he will want to put it into the structure that's most appropriate for his development setup.
